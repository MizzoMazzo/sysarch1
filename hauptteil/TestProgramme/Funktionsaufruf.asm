jal func
addiu $6, $0, 456
loop:
j loop

func:
addiu $5, $0, 123
jr $ra