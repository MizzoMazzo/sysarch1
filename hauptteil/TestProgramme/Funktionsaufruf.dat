0x0c100003
0x240601c8
0x08100002
0x2405007b
0x03e00008
