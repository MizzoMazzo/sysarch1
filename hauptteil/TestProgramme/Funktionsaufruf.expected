@5 7b
@6 1c8
@1f 00400004
